README.md

```diff
@@ -30,6 +31,29 @@ The code in this repository is licensed the Apache 2.0 license unless otherwise
 noted. Please see the [LICENSE file](https://github.com/edx/ux-pattern-
 library/blob/master/LICENSE.txt) for details.
 
+## Dependencies
+
+Using the edX Pattern Library source code in a project current requires the following:
+
+* Your project use and can compile Sass into CSS using perferrably [LibSass](http://sass-lang.com/libsass). The edX Pattern Library currently does not provide pre-compiled CSS files.
+* You to locally install [Bower](http://bower.io/) - [see below](#development---ux-pattern-library-package) for more installation details.
+* Your project's Sass manually calls ``edx-pattern-library`` Sass partials in its main compile file.
```

Comment:
I would suggest either altering the main sentence, or the first 3 bullet points to make the list context clearer. 

"project current(ly*) requires the following:" (of the 4 line items that follow, the last is clearest given this heading.) 

You might be able to get away with simply dropping "the following" (with some slight adjustment to the start of some of the bullets.